src/ffi/logging.rs

```diff
@@ -310,9 +364,62 @@ mod test {
     }
 
     #[test]
-    fn slow_telio_cb_handling() {
-        const LOGS_TO_DROP: usize = 5;
+    fn test_trace_via_slow_telio_cb() {
```

Comment:
I would like to see a test there we call a stopper in middle of an act, as we are not testing functionality actually used by apps. as FfiCalback drop will not be practically used then using global logger: (see [example]( https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f7a8eed276096ff8f62c8819f5c4aa40) )

But it's used then using with_default.

Reply:
I think the update `test_logger` integration test does that.